### PR TITLE
Fix webpack config and some assets URLs

### PR DESCRIPTION
**Pull request by kravets-levko**

### CODE REVIEW

File: webpack.config.js [hunk 1]
```diff
@@ -143,10 +143,14 @@ const config = {
...
-      { from: /./, to: '/static/index.html' }
-    ]},
-    contentBase: path.join(__dirname, 'client', 'app'),
+    index: '/static/index.html',
+    historyApiFallback: {
+      index: '/static/index.html',
+      rewrites: [
+        { from: /.?/, to: '/static/index.html' },
+      ],
+    },
```
Comment (arikfr) [inline webpack.config.js]: Maybe this is not needed anymore with the `index` setting?

Comment (kravets-levko) [reply webpack.config.js]: no, I tried - it does not work without it. actually, I don't understand why, but it does not work.

Comment (kravets-levko) [reply webpack.config.js]: hm, well, I'm wrong, it works without this line. will remove it

**Comment by arikfr** [comment]
💃

**Pull request merged**